src/review/routes/profile/Details.js

```diff
@@ -1,20 +1,32 @@
 import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import Text from 'material-ui/Text'
+import Button from 'material-ui/Button'
+import Link from 'modules/components/Link'
 import ViewContainer from 'modules/components/ViewContainer'
 import ScrollView from 'modules/components/ScrollView'
 import LayoutBody from 'modules/components/LayoutBody'
 import ReviewAppBar from 'review/modules/AppBar/AppBar'
 
-function ProfileDetails(props) {
+function ProfileDetails({ user }) {
   return (
     <ViewContainer>
       <ReviewAppBar />
       <ScrollView>
         <LayoutBody margin>
           <Text type="display1" component="h2" gutterBottom>
-            {props.user.name}
+            {user.name}
           </Text>
+          {!user.privateSync && (
```

Comment:
But you are right, it can be useful because if you revoke and retry, I don't think I handle it.